ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,48 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
+  const { download, deleteDownloadModel, exporationExport } = props;
+
+  const expireExports = moment(download.get('time')).add(exporationExport.get('ttl'), 'seconds');
```

Comment:
As explained above, this will all change to use an `expiresAt` date _on_ the download model itself, which should mean we don't need to worry about accessing the expirationExport settings at all